gradle/test.gradle

```diff
@@ -168,26 +168,31 @@ subprojects {
   }
 
   task repeatDistributedTest(type: RepeatTest) {
+    dependsOn {distributedTest.getTaskDependencies()}
```

Comment:
A trivial sort of thing, but I believe accessing fields via property rather than getter, e.g. `distributedTest.taskDependencies`, is the more typical Groovy style.